Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN make -C "tools"
 
 # Final
 FROM golang:1.12.5-stretch as final
-
+ARG builder_uid
+RUN adduser --disabled-password --gecos "" --uid ${builder_uid} builder
```

Comment:
This commit actually introduced a regression, where if users have `sudo` required to run `docker` commands, their build will fail on this line. I've sent https://github.com/digitalbitbox/bitbox-base/pull/99 to revert this commit for now, as we figure out the best path forward.